src/redux/actions/ws.js

```diff
@@ -9,8 +10,13 @@ const send = payload => ({
     : JSON.stringify(payload),
 })
 
+const getScope = () => {
```

Comment:
It shouldn`t be in actions

Reply:
where do you think it would be a better place? I'm not so familiar with the code organization on this repo

Reply:
maybe in the [config](https://github.com/tarcisiozf/bfx-hf-ui-core/blob/external-dms/src/redux/config.js), where the isElectronApp constant is being set?

Reply:
updated: d443173290aa86a2c563e0475e03a998e25552e7